pumpkin-cli/tests/mzn_constraint_test.rs

```diff
@@ -13,6 +14,47 @@ macro_rules! mzn_test {
     };
 }
 
+macro_rules! cumulative_test {
```

Comment:
@maartenflippo - Create a general macro for running with options and then create a cumulative macro